packages/raml-generator/package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "@commercetools-docs/ramldoc-generator",
+  "version": "1.0.0",
+  "description": "Generates RAML docs compatible with @commercetools-docs/gatsby-transformer-raml",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "keywords": [
+    "cli",
+    "ramldoc-generator"
+  ],
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "bin": {
+    "commercetools-docs-ramldoc-generator": "bin/ramldoc-generator.js"
```

We can name the binary like this (we do the same with the broken link checker):

```suggestion
    "commercetools-ramldoc-generator": "bin/ramldoc-generator.js"
```

---

I don't understand why this is necessary.

---

Reasons:
- consistency with the other existing binaries
- shorter name
- unnecessary information in the name, the `commercetools` prefix is enough

---

Reasons:
- consistency with the other existing binaries
- shorter name
- unnecessary information in the name, the `commercetools` prefix is enough

---

So why do we have "commercetools-docs" scope or namespace instead of "commercetools"?

---

The npm/package.json scope has nothing to do with the binary name